themes/negligible.omp.json

```diff
@@ -0,0 +1,82 @@
+{
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh3/main/themes/schema.json",
+    "blocks": [
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "segments": [
+                {
+                    "type": "os",
+                    "style": "powerline",
+                    "foreground": "cyan"
```

Comment:
It's prettier when this one has the `"prefix": ""` property. Not sure if this is on purpose, if so I'm inclined to merge, just my two cents as without that there's a space before the OS icon starts.

Reply:
Thanks for mentioning that, I will fix that